### ISSUE TITLE
Sponsor API

### ISSUE THREAD

**Issue by josselinlbe** [junhoyeo/threads-api#29]
Hey @junhoyeo ✌ we've integrated a similar (commercial) API at [Piloterr](https://docs.piloterr.com/v2/api-reference/threads/user/info)! :) Companies can sponsor the project?

**Comment by junhoyeo** [comment]
Absolutely, @josselinlbe! I'm eagerly awaiting the approval for GitHub Sponsors that I requested a few hours ago. I'm very open to sponsorships, and your support would indeed be invaluable. 😉
I'll let you know when it's live! Also, feel free to reach out to me at [i@junho.io](mailto:i@junho.io) if you prefer!

**Comment by iamdeepak92** [comment]
This is the first Threads scraper code. This requires no login . I think this will help someone.

https://github.com/iamdeepak92/threads-scraper
